package.json

```diff
@@ -26,6 +26,10 @@
     "packages/*",
     "apps/*"
   ],
+  "overrides": {
```

Comment:
is this necessary?

Reply:
React-native-shake use react-native 72 instead of 73 as a devdependency and it would break whole app during build so I enforce it to use 0.73

It looks like it works fine but if we want to remove it we need to remove the library, and the shake feature or find some different library to detect the gesture